### PR TITLE
test for JAX class injecting into path

### PR DESCRIPTION
**Pull request by benjamin-confino**

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

File: dev/com.ibm.ws.cdi.jee_fat/fat/src/com/ibm/ws/cdi12/fat/tests/InjectIntoPath.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
+/*******************************************************************************
+ * Copyright (c) 2014, 2018 IBM Corporation and others.
```
Comment (hutchig) [inline dev/com.ibm.ws.cdi.jee_fat/fat/src/com/ibm/ws/cdi12/fat/tests/InjectIntoPath.java:2]: , 2020 methinks

File: dev/com.ibm.ws.cdi.jee_fat/fat/src/com/ibm/ws/cdi12/fat/tests/InjectIntoPath.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        return null;
+    }
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+
+        WebArchive injectIntoPathApp = ShrinkWrap.create(WebArchive.class, "injectIntoPath.war")
+                        .addPackage(com.ibm.injectintopath.CDIResource.class.getPackage());
+
+        server = LibertyServerFactory.getLibertyServer("jaxInjectIntoPathServer");
```
Comment (hutchig) [inline dev/com.ibm.ws.cdi.jee_fat/fat/src/com/ibm/ws/cdi12/fat/tests/InjectIntoPath.java:61]: why does the server start with 'jax' and not something like cdi?

**Review by hutchig — APPROVED** [review]
One or two trivial comments but looks good to me if those are considered

**Comment by benjamin-confino** [comment]
last update was just a copyright comment

#run-libby-bot

**Pull request merged**